recipes/armadillo/all/conandata.yml

```diff
@@ -0,0 +1,21 @@
+sources:
+  "10.6.1":
```

Comment:
Do you require the older releases? If not can we please remove them? https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#what-is-the-policy-for-adding-older-versions-of-a-package

Reply:
Nah I guess not. I just added the newer releases because I saw they were available. I'll update this PR to be for 10.7.0